WasatchNET/EEPROM.cs

```diff
@@ -1879,15 +1915,12 @@ protected bool writeParse()
 
                 if (subformat == PAGE_SUBFORMAT.HANDHELD_DEVICE)
                 {
-                    if (!ParseData.writeByte(libraryType, pages[7], 0)) return false;
-                    if (!ParseData.writeUInt16(libraryID, pages[7], 1)) return false;
-                    if (!ParseData.writeByte(startupScansToAverage, pages[7], 3)) return false;
-
-                    if (subformat != PAGE_SUBFORMAT.HANDHELD_DEVICE)
-                    {
-                        logger.error("EEPROM.writeLibrary inapplicable on subformat 0x{0:2x}", subformat);
-                        return false;
-                    }
+                    if (!ParseData.writeByte(libraryType,               pages[7], 0)) return false;
+                    if (!ParseData.writeUInt16(libraryID,               pages[7], 1)) return false;
+                    if (!ParseData.writeByte(startupScansToAverage,     pages[7], 3)) return false;
+                    if (!ParseData.writeByte(matchingMinRampPixels,     pages[7], 4)) return false;
+                    if (!ParseData.writeUInt16(matchingMinPeakHeight,   pages[7], 5)) return false;
+                    if (!ParseData.writeByte(matchingThreshold,         pages[7], 7)) return false;

```

Comment:
I need to review the library features. These presuppose a large library of EEPROM pages though, correct?

Reply:
The "library" is essentially a stack of 61 EEPROM pages (zero-indexed 10 through 70), enough to store 61 x 64 = 3904 bytes of data, i.e. 1952 uint16, or the size of one SiG spectrum.  I'll send you a link with additional context.